docs/manuals/source/IntroductionAndTutorial/InstallingBareosWebui.rst

```diff
@@ -440,6 +440,69 @@ As an alternative to the method above the Bvfs cache can be updated after each j
 
    We do provide a specific JobId in the *JobId* command argument in this example. Only the *JobId* given by the placeholder %i will be computed into the cache.
 
+Upgrade from 21.x.x to 22.x.x
+-----------------------------
+
+|webui| now requires PHP-FPM instead of Apache2 mod-php. Due to this change there are some manual steps to consider
+before and after upgrading. Please have a look at the following remarks according to your operating system of choice.
```

Comment:
```suggestion
|webui| now requires **php-fpm** instead of apache2 **mod-php**. Due to this change there are some manual steps to consider. Before upgrading, apache mod-php needs to be disabled or removed. After upgrading, php-fpm needs to be enabled. For details please check he following remarks according to your operating system of choice.
```

Reply:
Applied, please see https://github.com/bareos/bareos/pull/1287/commits/da48955442b440c0892f8862c65fa795c4a23872 .